src/08_comprehensions.py

```diff
@@ -11,14 +11,16 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
-
+for x in range(1,6):
```

Comment:
list comprehension is a specific python syntax. 
You could write all of this on one line with `y = [x for x in range(1,6)]`

I recommend this article: 
https://www.geeksforgeeks.org/python-list-comprehension-and-slicing/